app/models/vacancy.rb

```diff
@@ -300,6 +300,22 @@ def is_a_teaching_or_middle_leader_role?
     job_roles.intersect?(%w[teacher head_of_year_or_phase head_of_department_or_curriculum sendco other_leadership])
   end
 
+  def new_application_path
+    if has_uploaded_form?
+      Rails.application.routes.url_helpers.jobseekers_job_job_application_path(id)
+    else
+      Rails.application.routes.url_helpers.new_jobseekers_job_job_application_path(id)
+    end
+  end
+
+  def has_uploaded_form?
+    receive_applications == "uploaded_form"
+  end
```

Comment:
`enum` auto defines helper methods, so you can `vacancy.uploaded_form?` instead of defining has_uploaded_form?